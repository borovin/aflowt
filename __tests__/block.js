import test from 'ava'
import Browser from './helpers/browser'
import {prettyPrint} from 'html'

test.beforeEach(t => {
  t.context.browser = new Browser()

  return t.context.browser.goto('/examples/block.html')
})

test.afterEach(t => {
  return t.context.browser.close()
})

test('Test block example', async t => {
  const snapshot = await t.context.browser.snapshot('b-test')

  t.snapshot(prettyPrint(snapshot))
})
