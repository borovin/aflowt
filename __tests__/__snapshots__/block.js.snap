// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test block example 1`] = `
"<b-test a=\\"\\" b=\\"\\" d=\\"0\\" e=\\"1\\" f=\\"1.2\\" g=\\"string\\">this.a: true:boolean
    <br>this.b: true:boolean
    <br>this.c: null:object
    <br>this.d: 0:number
    <br>this.e: 1:number
    <br>this.f: 1.2:number
    <br>this.g: string:string
    <br>this.h: false:boolean
    <br>
</b-test>"
`;
